src/app/technical-record-search/technical-record-search.module.ts

```diff
@@ -27,11 +26,7 @@ import { VehicleTechRecordModelEffects } from '@app/store/effects/VehicleTechRec
     HttpClientModule,
     RouterModule.forChild([
       {path: '', component: TechnicalRecordSearchComponent, canActivate: [AuthenticationGuard]},
-      {
```

Comment:
add routes in a separate file. Follow approach similar to the following link: https://angular.io/guide/lazy-loading-ngmodules

Reply:
We would need the routes once you move "create" component over here.

Reply:
It should be similar to:

```
import { Routes, RouterModule } from '@angular/router';

import { TechnicalRecordComponent } from './PATH_TO_COMPONENT';
import { TechnicalRecordCreateComponent } from './PATH_TO_COMPONENT';


const routes: Routes = [
  {
    path: '',
    component: TechnicalRecordComponent
  },
   {
     path: 'create',
    component: TechnicalRecordCreateComponent
  }
];
export class TechnicalRecordRoutingModule { }
```